contents/blog/startup-ops-toolkit.md

```diff
@@ -0,0 +1,93 @@
+
+### Contractors
+
+<a target="_blank" rel="noopener noreferrer nofollow" href="https://www.letsdeel.com/">Deel</a> is the only way to go. They cover pretty much any country and currency you’ll need, handle all the compliance stuff for you, and generally make the process way more pleasant than it has any right to be. Plus their support is insanely responsive and helpful. It’s probably my favourite ops product. Deel charge $35/month per contractor.
+
+If you need an Employer of Record service, Deel now offer this too - super useful if you are hiring permanent employees outside the US in a country where you don’t have a legal presence.
+
+### Contracts
+
+Orrick have a <a target="_blank" rel="noopener noreferrer nofollow" href="https://www.orrick.com/en/Total-Access/Tool-Kit/Start-Up-Forms/Employment-and-Consultant">handy free set of templates</a> you can use! (US only)
+
+You need both an Offer Letter and a Confidential Information and Invention Assignment Agreement. Deel have free templates integrated into their platform already, so you’ll only usually use these for permanent employees, unless you want to use a customized contractor agreement in Deel.
+
+Do not pay a lawyer to create contract templates for you - it is a fantastic waste of money better spent when you need them to handle your next funding round.
+
+
+## Managing your money (so you can keep paying people)
+
+There are basically two sides to finance - accounting and budgeting. You know this already from YC’s <a target="_blank" rel="noopener noreferrer nofollow" href="https://www.startupschool.org/">Startup School</a>, of course.
+
+<iframe src="https://giphy.com/embed/o2xbqBmHbQAbm" width="480" height="269" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/move-donald-trump-o2xbqBmHbQAbm">via GIPHY</a></p>
+
+### Accounting
+
+For your accounting, you basically want to do the bare minimum to be legally compliant and not sued by the IRS; and have financial reports to look at so you can nod and go ‘hm, yes we are still burning money at an alarming rate.’
+
+We use <a target="_blank" rel="noopener noreferrer nofollow" href="https://www.tryfondo.com/">Fondo<a/> and they are unbelievably good. They basically act as an outsourced accounting team, and you submit everything through a shared Slack channel with them or via an online portal, which is remarkably straightforward to use. Fondo will handle your tax returns, annual filing and also send you monthly management reports. They charge on a sliding scale based on your expenses, so I’m not sharing what we pay here for confidentiality reasons.
```

Comment:
this link is breaking - @seanpackham perhaps try rebuilding with fewer rel options? Weird
![Uploading Screenshot 2021-03-23 at 11.55.57.png…]()

Reply:
Fixed!